docs/userGuide/troubleshooting.md

```diff
@@ -40,20 +37,57 @@ This is likely caused by incorrect HTML markup, for example nesting block-level
 or missing `<tbody>`.
 Bailing hydration and performing full client-side render.
 ```
+
 See [SSR guide for Vue](https://vuejs.org/guide/scaling-up/ssr.html#hydration-mismatch) for more details on hydration mismatch.
 </panel>
 
 A possible fix for the above situation is to wrap the table in a `<div>` element instead:
 
 ```html
-
 <div id="example">
-
-Animal | Trainable? | Price | Remarks
-:------|:----------:|------:|--------
-Ants   |     no     |     5 |
-Bees   |     no     |    20 |
-Cats   |    yes     |   100 |
+  Animal | Trainable? | Price | Remarks :------|:----------:|------:|--------
+  Ants | no | 5 | Bees | no | 20 | Cats | yes | 100 |
 </div>
 ```
 
+##### Markdown Rendering Issues
+
+If you encounter issues of failure in rendering markdown in a component, it is likely that the markdown is not being properly recognized due to syntax errors. Sign posting is required to inform markdown to parse content of a presentation component as markdown rather than plain text.
+
+You could sign post markdown either by:
+
+- using the `markdown` tag
+- using an empty line without any indentation before the markdown content
+
+<panel header="Markdown rendering (Example)" type="seamless">
```

Comment:
Can use the codeAndOutput boiler plate for this part to be more consistent! :)
Also maybe can add an example of using empty line?

Reply:
Noted. 
Thank you for the suggestion, requested changes made.